neo/Network/RPC/RpcServer.cs

```diff
@@ -488,7 +488,7 @@ private JObject GetPeers()
             {
                 JObject peerJson = new JObject();
                 peerJson["address"] = p.Remote.Address.ToString();
-                peerJson["tcpPort"] = p.ListenerTcpPort;
+                peerJson["port"] = p.ListenerTcpPort;
```

Comment:
We could specify that is ´Tcp`

Reply:
I think `getpeers` returns Tcp peers, the ports must be tcp, no need to specify it.